docs/ui/auth/fragments/react/sign-out.md

```diff
@@ -1,4 +1,4 @@
-<amplify-sign-out button-text="Custom Text"></amplify-sign-out>
+<amplify-sign-out buttonText="Custom Text"></amplify-sign-out>
 
 **Usage**
 
```

Comment:
Could you also make the same change down on line 6?  GitHub is preventing me from commenting directly on that line for some reason...